docs/run-conformance-tests.md

```diff
@@ -3,16 +3,30 @@ title: Conformance tests
 weight: 50
```

Comment:
>docs: Fix the conformance tests formatting & grammer

What is `grammer` ? :smile:

Reply:
Yeah I don't remember either. But getting rid of this. Since this commit has evolved than what it was meant to do.